src/plot_circulareegtopoplot.jl

```diff
@@ -113,7 +113,7 @@ function calculateGlobalMaxValues(data, predictor)
     )
 
 
-    globalMaxVal = maximum(x.localMaxVal)
+    globalMaxVal = round(maximum(x.localMaxVal), digits = 2)
```

Comment:
I dont think this is a good idea. why? what if localMaxVal is a very small number?

Reply:
You mentioned some approximation function
Is it ApproxFun from https://github.com/unfoldtoolbox/UnfoldSim.jl/pull/70

Reply:
the function is literally called `approx` - maybe googling "Floating point comparison approx julia` or something can give you some context. 

I think you need approx also not here, but in the unit test, correct?